### ISSUE TITLE
Choose alternative text value based on language

### ISSUE THREAD

**Issue by petli** [commonsmachinery/libcredit#12]
See https://github.com/commonsmachinery/credit-metadata-best-practices/blob/master/parsing-metadata.md#alternative-text-values

**Comment by artfwo** [comment]
In both Python and JS worlds we pass the i18n/gettext instance that provides language info as an argument to format(), while the RDF data is parsed in Credit constructor. I think it's okay to pass i18n instance to the constructor here, but that breakes our current API. Ok to break it?

**Comment by petli** [comment]
Can we get the language code out from the i18n object?  I don't find any method in the API for jed.js to do it.

Allowing caller of credit constructor to pass in a list of language codes in preferred order would be easier for us, code-wide, and I don't think too horrible for users.

**Comment by artfwo** [comment]
Yes, language code is stored in locale_data dict in JS and gettext.info in Python.

**Comment by petli** [comment]
It doesn't appear to be an official API, though.  Another concern is that the locale might be e.g. es_AR, but any spanish-language title would do.
